guides/tutorials/codelabs/azure-vmss-source-to-prod/index.md

```diff
@@ -0,0 +1,326 @@
+  4. The deployment is verified
+  5. Spinnaker redeploys the same image to the production environment and disable the previous deployment.
+  
+This codelab will walk through the steps to  implement this scenario: 
+
+  * Setup the environment and the necessary pre-requisites
+  * Configure Jenkins and Spinnaker
+  * Create the pipeline for the deployment
+
+## 0. Environment pre-requisites 
+
+### Deploy Spinnaker and Jenkins on Azure 
+
+You need the following resources to perform the codelab: 
+- **Azure subscription**: You can create one for free using the [Azure Free trial](https://azure.microsoft.com/free/).
+- **Spinnaker and Jenkins VM**: You can deploy them on your Azure subscription with the [Continuous Deployment quick start template](https://github.com/Azure/azure-quickstart-templates/tree/master/301-jenkins-aptly-spinnaker-vmss) 
+
+From a shell run the following command that will walk you through all the steps necessary to deploy Jenkins and Spinnaker in Azure:
+
+```bash 
+bash <(curl -sL https://aka.ms/DeployDevOps)
+```
+
+In the output of the above script you will find the command to open an SSH tunnel to the DevOps VM that has just been deployed. The command should look like this: 
+
+```bash
+ssh -L 8080:localhost:8080 -L 9000:localhost:9000 -L 8084:localhost:8084 -L 8087:localhost:8087 username@vmdnsprefix.selected_region.cloudapp.azure.com
+```
+
+> NOTE: You will have to enter the password that you have specified in the previous step.
+
+
+## 1. Configure Jenkins 
+
+1. After you have started your ssh tunnel, navigate to `http://localhost:8080/` on your local machine.
+1. The instance should already be unlocked and your first account setup. Login with the credentials you specified when deploying the template.
+1. Click **Install suggested plugins**: 
+   ![Install Jenkins plugins](images/1-Install-Jenkins-plugins.png)
+1. Your Jenkins instance is now ready to use!
+1. Run the `Sample Aptly Job` by clicking on the run button for the job. Verify that the job completes successfully.
+
+   ![Run Jenkins job](images/1-run-sample-aplty-job.png)
+
+## 2. Configure Spinnaker
+
+### I. Create the "codelab" application 
+
+1. Navigate to your Spinnaker VM by using `http://localhost:9000` on your local machine. 
+1. Click on the **Actions** menu on the right
+
+   ![Actions](images/2-actions-create-applications.png)
+   
+1. In the *New Application* window, enter the following values: 
+  * Name: codelab
+  * Owner Email: Enter your email
+  
+    And click **Create**
+  
+    ![New Application](images/2-create-application.png)
+  
+    You can read more how the Spinnaker application matches to Azure resources in the [Azure reference for Spinnaker](/reference/providers/azure.md)
+
+### II. Create a Security Group"
+
+1. Navigate to the [SECURITY GROUPS](http://localhost:9000/#/applications/codelab/securityGroups) page and click on the **Create Security Group** button.

```

Comment:
I always hate all caps stuff in articles because it feels like it's yelling at me :)